src/org/javarosa/core/model/ItemsetBinding.java

```diff
@@ -55,18 +59,27 @@
     private List<SelectChoice> choices; //dynamic choices -- not serialized, obviously
 
     public boolean randomize = false;
-    public Long randomSeed = null;
+    public XPathNumericLiteral randomSeedNumericExpr = null;
```

Comment:
Not a show stopper, but aren’t these initializers redundant?

Reply:
I guess they look like redundant, although IntelliJ isn't highlighting them for me... Are you confident that we can remove them?

Reply:
I, also, like the reassurance of a message from IDEA. I’m too lazy to look up field initialization in the JLS right now, and I’m not gonna go out on a limb. :-)

Reply:
Let's leave it then :)